src/Catalyst.Core.Lib.Tests/IntegrationTests/Validators/TransactionValidatorTests.cs

```diff
@@ -71,12 +68,12 @@ public void ValidateTransactionSignature_will_pass_with_valid_transaction_signat
             var signature = new Signature
             {
                 // sign an actual TransactionBroadcast object
-                RawBytes = cryptoContext.Sign(privateKey, validTransactionBroadcast.ToByteArray(), signingContext.ToByteArray())
+                RawBytes = cryptoContext.Sign(privateKey, validTransactionBroadcast.PublicEntry.ToByteArray(), signingContext.ToByteArray())
```

Comment:
Since the mempool item/publicEntry is now flat and not a batch of transactions we can refactor most of the tests and build just the PublicEntry and avoid the TransactionBroadcast.